__tests__/app.test.js

```diff
@@ -41,4 +41,46 @@ describe('GET /api', () => {
             expect(response.body).toEqual(endpoints);
         });
     });
+});
+
+
+describe('GET /api/articles/:article_id', () => {
+    test('returns article object with correct properties', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then((response) => {
+            const article = response.body.article;
+
+            expect(typeof article).toBe('object');
+
+            expect(article).toMatchObject({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
```

Comment:
We do want to check article_id property is correct Id. For instance .get("/api/articles/1") should have an article_id property with value being 1